includes/definitions/cyberpower.yaml

```diff
@@ -0,0 +1,13 @@
+os: cyberpower
+text: 'Cyberpower'
+type: power
+icon: cyberpower
+over:
+    - { graph: device_current, text: Current }
+    - { graph: device_voltage, text: Voltage }
+discovery:
+    - sysObjectId: .1.3.6.1.4.1.3808.1.1.3
+poller_modules:
+    ports: 0
```

Comment:
Nothing usable. SNMP for it is effectively broken when it comes to that. It provides one interface and the stats never change.